accel_cov:
- - 0
  - 1
  - 2
- - 3
  - 4
  - 5
- - 6
  - 7
  - 8
accel_offset:
- 3
- 3
- 3
gyro_cov:
- - 0.001
  - 0
  - 0
- - 0
  - 0.002
  - 0
- - 0
  - 0
  - 0.003
gyro_offset:
- 4
- 4
- 4
odom_pose_cov:
- - 0.0001
  - 0
  - 0
  - 0
  - 0
  - 0
- - 0
  - 0.0002
  - 0
  - 0
  - 0
  - 0
- - 0
  - 0
  - 0.0003
  - 0
  - 0
  - 0
- - 0
  - 0
  - 0
  - 0.0004
  - 0
  - 0
- - 0
  - 0
  - 0
  - 0
  - 0.0005
  - 0
- - 0
  - 0
  - 0
  - 0
  - 0
  - 0.006
odom_twist_cov:
- - 0.0006
  - 0
  - 0
  - 0
  - 0
  - 0
- - 0
  - 0.0005
  - 0
  - 0
  - 0
  - 0
- - 0
  - 0
  - 0.0004
  - 0
  - 0
  - 0
- - 0
  - 0
  - 0
  - 0.0003
  - 0
  - 0
- - 0
  - 0
  - 0
  - 0
  - 0.0002
  - 0
- - 0
  - 0
  - 0
  - 0
  - 0
  - 0.0001